apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.scss

```diff
@@ -51,6 +52,7 @@
 
   .card {
     height: 15.8125rem;
+    height: auto !important;
```

Comment:
Can you try updating the cards in the design system instead so we can avoid `!important`